api/metadata/models.py

```diff
@@ -0,0 +1,44 @@
+from core.models import AbstractBaseExportableModel
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+
+from organisations.models import Organisation
+
+FIELD_DATA_MAX_LENGTH = 2000
+
+
+class FieldType(models.TextChoices):
+    INTEGER = "int"
+    STRING = "str"
+    BOOLEAN = "bool"
+    FLOAT = "float"
+
+
+class MetadataField(AbstractBaseExportableModel):
```

Comment:
So, as I understand it: 

This is a class for creating generic fields in an organisation?

...

Reply:
Yep